xtevent/xtevent.sthlp

```diff
@@ -146,40 +151,13 @@ is the number of periods before the event.
 {opt overidpre} and 
 {opt overidpost} can be declared. 
 
-{phang} {opth norm(integer)} specifies the event-time coefficient to be normalized to 0.
-The default is to normalize the coefficient on -1.
-
 {phang}
-{opth proxy(varlist)} specifies proxy variables for the confound to be included. {opt proxy} is not allowed with {opt cohort}, 
-{opt control_cohort} or {opt sunabraham}.
-
-{marker proxyiv_spec}{...}
-{phang}
-{opt proxyiv(proxyiv_spec)} specifies instruments for the proxy variable for the policy. {cmd:proxyiv()} admits three syntaxes to use 
-either leads of the policy variable or additional variables as instruments. {opt proxy} is not allowed with {opt cohort}, 
-{opt control_cohort} or {opt sunabraham}. 
-
-{phang2}
-{cmd:proxyiv(select)} is the default and selects the lead with the strongest first stage among all possible leads of the 
-differenced policy variable to be used as an instrument.
-{cmd:proxyiv(select)} is the default for the one proxy, one instrument case, and it is only available in this case. 
-
-{phang2}
-{cmd:proxyiv(# ...)} specifies a numlist with the leads of the differenced policy variable as instruments. For example, 
-{cmd:proxyiv(1 2)} specifies that the two first leads of the difference of the policy variable will be used as instruments.
-
-{phang2}
-{opth proxyiv(varlist)} specifies a {it:varlist} with the additional variables to be used as instruments.
-
-{phang}
-{opt nofe} excludes panel fixed effects.
-
-{phang}
-{opt note} excludes time fixed effects.
+{opt static} estimates a static panel data model and does not generate or plot event-time dummies. {opt static} is not allowed with 
+{opt window}, {opt pre}, {opt post}, {opt overidpre}, {opt overidpost}, or {opt diffavg}.
 
 {marker imputetype}{...}
 {phang}
-{opt impute(type, [ saveimp])} imputes missing values in {it:policyvar} and uses this new variable as the actual {it:policyvar}. 
+{opt impute(type, [ saveimp])} imputes leads, lags, and missing values of {it:policyvar} and uses this new variable as the actual {it:policyvar}. 

```

Comment:
@jorpppp 
What do you think of adding here in the impute section a suggestion for the user to check the [impute note](https://rawcdn.githack.com/JMSLab/xtevent/cf16d12f90ddf363df62c397cf0e9dc05bbd9875/impute_option_description.html)?

Reply:
implemented in https://github.com/JMSLab/xtevent/commit/d293178989e270af5d1bbc4dc41a9c49c1a9337b

Reply:
Thanks, don't forget to edit the impute note too (we should do it in this PR).